CVSROOT:	/cvs/mesa
Module name:	Mesa
Repository:	Mesa/src/mesa/x86-64/
Changes by:	idr@gabe.	05/08/03 16:05:26

Log message:
  Convert all instances of XTHREADS to USE_XTHREADS.  This fixes one of
  serveral things that are broken when building on a system with X.org 7.0rc0
  installed.

Modified files:
      Mesa/src/glx/x11/:
        glxclient.h glxext.c 
      Mesa/src/mesa/x86/:
        glapi_x86.S 
      Mesa/src/mesa/glapi/:
        gl_x86-64_asm.py gl_x86_asm.py glthread.c glthread.h 
      Mesa/src/mesa/x86-64/:
        glapi_x86-64.S 
  
  Revision      Changes    Path
  1.13          +4 -4      Mesa/src/glx/x11/glxclient.h
  http://cvs.freedesktop.org/mesa/Mesa/src/glx/x11/glxclient.h
  1.16          +2 -2      Mesa/src/glx/x11/glxext.c
  http://cvs.freedesktop.org/mesa/Mesa/src/glx/x11/glxext.c
  1.55          +1 -1      Mesa/src/mesa/x86/glapi_x86.S
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/x86/glapi_x86.S
  1.2           +1 -1      Mesa/src/mesa/glapi/gl_x86-64_asm.py
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/glapi/gl_x86-64_asm.py
  1.14          +1 -1      Mesa/src/mesa/glapi/gl_x86_asm.py
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/glapi/gl_x86_asm.py
  1.12          +1 -1      Mesa/src/mesa/glapi/glthread.c
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/glapi/glthread.c
  1.21          +3 -3      Mesa/src/mesa/glapi/glthread.h
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/glapi/glthread.h
  1.2           +1 -1      Mesa/src/mesa/x86-64/glapi_x86-64.S
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/x86-64/glapi_x86-64.S

Index: src/glx/x11/glxclient.h
===================================================================
RCS file: /cvs/mesa/Mesa/src/glx/x11/glxclient.h,v
retrieving revision 1.12
retrieving revision 1.13
diff -u -b -B -r1.12 -r1.13
--- Mesa-6.3.1.1/src/glx/x11/glxclient.h	1 Aug 2005 16:30:24 -0000	1.12
+++ Mesa-6.3.1.1/src/glx/x11/glxclient.h	3 Aug 2005 23:05:25 -0000	1.13
@@ -60,7 +60,7 @@
 #include "GL/internal/glcore.h"
 #include "glapitable.h"
 #include "glxextensions.h"
-#if defined( XTHREADS )
+#if defined( USE_XTHREADS )
 # include <X11/Xthreads.h>
 #elif defined( PTHREADS )
 # include <pthread.h>
@@ -547,7 +547,7 @@
 extern int __glXDebug;
 
 /* This is per-thread storage in an MT environment */
-#if defined( XTHREADS ) || defined( PTHREADS )
+#if defined( USE_XTHREADS ) || defined( PTHREADS )
 
 extern void __glXSetCurrentContext(__GLXcontext *c);
 
@@ -570,14 +570,14 @@
 #define __glXGetCurrentContext()	__glXcurrentContext
 #define __glXSetCurrentContext(gc)	__glXcurrentContext = gc
 
-#endif /* defined( XTHREADS ) || defined( PTHREADS ) */
+#endif /* defined( USE_XTHREADS ) || defined( PTHREADS ) */
 
 
 /*
 ** Global lock for all threads in this address space using the GLX
 ** extension
 */
-#if defined( XTHREADS )
+#if defined( USE_XTHREADS )
 extern xmutex_rec __glXmutex;
 #define __glXLock()    xmutex_lock(&__glXmutex)
 #define __glXUnlock()  xmutex_unlock(&__glXmutex)
Index: src/glx/x11/glxext.c
===================================================================
RCS file: /cvs/mesa/Mesa/src/glx/x11/glxext.c,v
retrieving revision 1.15
retrieving revision 1.16
diff -u -b -B -r1.15 -r1.16
--- Mesa-6.3.1.1/src/glx/x11/glxext.c	1 Aug 2005 16:30:24 -0000	1.15
+++ Mesa-6.3.1.1/src/glx/x11/glxext.c	3 Aug 2005 23:05:25 -0000	1.16
@@ -145,7 +145,7 @@
  * Current context management and locking
  */
 
-#if defined( XTHREADS )
+#if defined( USE_XTHREADS )
 
 /* thread safe */
 static GLboolean TSDinitialized = GL_FALSE;
@@ -1106,7 +1106,7 @@
     XEDataObject dataObj;
     int major, minor;
 
-#if defined(XTHREADS)
+#if defined(USE_XTHREADS)
     {
         static int firstCall = 1;
         if (firstCall) {
Index: src/mesa/glapi/gl_x86-64_asm.py
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/glapi/gl_x86-64_asm.py,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -b -B -r1.1 -r1.2
--- Mesa-6.3.1.1/src/mesa/glapi/gl_x86-64_asm.py	2 Jul 2005 08:29:58 -0000	1.1
+++ Mesa-6.3.1.1/src/mesa/glapi/gl_x86-64_asm.py	3 Aug 2005 23:05:25 -0000	1.2
@@ -127,7 +127,7 @@
 		print '#  define HIDDEN(x)'
 		print '#endif'
 		print ''
-		print '#if defined(PTHREADS) || defined(XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)'
+		print '#if defined(PTHREADS) || defined(USE_XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)'
 		print '#  define THREADS'
 		print '#endif'
 		print ''
Index: src/mesa/glapi/gl_x86_asm.py
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/glapi/gl_x86_asm.py,v
retrieving revision 1.13
retrieving revision 1.14
diff -u -b -B -r1.13 -r1.14
--- Mesa-6.3.1.1/src/mesa/glapi/gl_x86_asm.py	21 Jun 2005 23:42:44 -0000	1.13
+++ Mesa-6.3.1.1/src/mesa/glapi/gl_x86_asm.py	3 Aug 2005 23:05:25 -0000	1.14
@@ -86,7 +86,7 @@
 		print '#define GLOBL_FN(x) GLOBL x'
 		print '#endif'
 		print ''
-		print '#if defined(PTHREADS) || defined(XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)'
+		print '#if defined(PTHREADS) || defined(USE_XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)'
 		print '#  define THREADS'
 		print '#endif'
 		print ''
Index: src/mesa/glapi/glthread.c
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/glapi/glthread.c,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -b -B -r1.11 -r1.12
--- Mesa-6.3.1.1/src/mesa/glapi/glthread.c	5 Jun 2003 00:50:40 -0000	1.11
+++ Mesa-6.3.1.1/src/mesa/glapi/glthread.c	3 Aug 2005 23:05:25 -0000	1.12
@@ -242,7 +242,7 @@
  * XFree86 has its own thread wrapper, Xthreads.h
  * We wrap it again for GL.
  */
-#ifdef XTHREADS
+#ifdef USE_XTHREADS
 
 unsigned long
 _glthread_GetID(void)
Index: src/mesa/glapi/glthread.h
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/glapi/glthread.h,v
retrieving revision 1.20
retrieving revision 1.21
diff -u -b -B -r1.20 -r1.21
--- Mesa-6.3.1.1/src/mesa/glapi/glthread.h	18 Jul 2005 12:31:26 -0000	1.20
+++ Mesa-6.3.1.1/src/mesa/glapi/glthread.h	3 Aug 2005 23:05:25 -0000	1.21
@@ -65,7 +65,7 @@
 
 
 #if (defined(PTHREADS) || defined(SOLARIS_THREADS) ||\
-     defined(WIN32_THREADS) || defined(XTHREADS) || defined(BEOS_THREADS)) \
+     defined(WIN32_THREADS) || defined(USE_XTHREADS) || defined(BEOS_THREADS)) \
     && !defined(THREADS)
 # define THREADS
 #endif
@@ -193,7 +193,7 @@
  * XFree86 has its own thread wrapper, Xthreads.h
  * We wrap it again for GL.
  */
-#ifdef XTHREADS
+#ifdef USE_XTHREADS
 #include <X11/Xthreads.h>
 
 typedef struct {
@@ -225,7 +225,7 @@
 #define _glthread_UNLOCK_MUTEX(name) \
    (void) xmutex_unlock(&(name))
 
-#endif /* XTHREADS */
+#endif /* USE_XTHREADS */
 
 
 
Index: src/mesa/x86/glapi_x86.S
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/x86/glapi_x86.S,v
retrieving revision 1.54
retrieving revision 1.55
diff -u -b -B -r1.54 -r1.55
--- Mesa-6.3.1.1/src/mesa/x86/glapi_x86.S	21 Jun 2005 23:42:44 -0000	1.54
+++ Mesa-6.3.1.1/src/mesa/x86/glapi_x86.S	3 Aug 2005 23:05:25 -0000	1.55
@@ -63,7 +63,7 @@
 #define GLOBL_FN(x) GLOBL x
 #endif
 
-#if defined(PTHREADS) || defined(XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)
+#if defined(PTHREADS) || defined(USE_XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)
 #  define THREADS
 #endif
 
Index: src/mesa/x86-64/glapi_x86-64.S
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/x86-64/glapi_x86-64.S,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -b -B -r1.1 -r1.2
--- Mesa-6.3.1.1/src/mesa/x86-64/glapi_x86-64.S	2 Jul 2005 08:29:58 -0000	1.1
+++ Mesa-6.3.1.1/src/mesa/x86-64/glapi_x86-64.S	3 Aug 2005 23:05:25 -0000	1.2
@@ -35,7 +35,7 @@
 #  define HIDDEN(x)
 #endif
 
-#if defined(PTHREADS) || defined(XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)
+#if defined(PTHREADS) || defined(USE_XTHREADS) || defined(SOLARIS_THREADS) || defined(WIN32_THREADS) || defined(BEOS_THREADS)
 #  define THREADS
 #endif
 
